extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -16,7 +16,16 @@
 
 import {isLayoutSizeDefined} from '../../../src/layout';
 import {user} from '../../../src/log';
+import {
+    installVideoManagerForDoc,
+} from '../../../src/service/video-manager-impl';
+import {VideoEvents} from '../../../src/video-interface';
+import {videoManagerForDoc} from '../../../src/services';
+import {assertAbsoluteHttpOrHttpsUrl} from '../../../src/url';
 
+/**
+ * @implements {../../../src/video-interface.VideoInterface}
```

Comment:
@honeybadgerdontcare Do you know what the best approach here is? We are adding a new attribute and using it in an example but looks like `test-example-validation` uses the deployed validator rather than local. Should we not use the new attribute in an example until validator is deployed?